data/remnant/remnant missions.txt

```diff
@@ -4479,3 +4479,140 @@ mission "Remnant: Expanded Horizons Astral job"
+			choice
+				`	"What do the storms have to do with the heat?"`
+				`	"What units do you use for temperature?"`
+					goto tempunits
+				`	"I think that is all the technical detail I can take."`
+					goto stormend
+			`	She looks mischevious "I can not tell you where or how we got it, but look at this." Her display shifts again with the hazes vanishing, and shows the galaxy map with one key difference. "Yes, the Ember Wastes are violet. Actually, they are off the visual spectrum into the ultraviolets, but this downshifts it so we can see it. This is what the Ember Wastes looked like a very long time ago.`
+			`	She takes on a serious tone. "This data suggests enough energy in the area to maintain the hydrogen at somewhere in excess of 61A8 degrees. It does not tell us the source of that energy, but based on patterns and some other data we have collected, the two appear to be intertwined. At the very least, the ion storms are one of the means by which energy is being transfered into the clouds." Her display vanishes and she slings her backpack over her shoulder.`
+				goto stormend
+			label numericsystem
+			`	She looks confused for a moment before responding. "It is called hexadecimal. It is a base 16 numeric system." She pauses to look at you. "Based on your reaction, I assume that base 10 is still the standard in human space?"`
+			choice
+				`	"Yes."`
+					goto  stormcontinue
+				`	"How does that work?"`
+			`	"Quite simply: We have added some extra characters so that when counting instead of a single digit position restarting at 0 every 10, they cycle every 16. So when we count we start at 1, but when we reach 9, the next number will be A. Then B, C, D, E and F. Then it is at that point that we add an extra digit and restart at 0 with 10. So in hexadecimal F would be equal to 15 in base 10. One of the advantages of using a numeric system with a higher base is that we can handle larger numbers with fewer digits. For example, with two digits we can communicate any integer up to what would be 255 in base 10. Or with three digits we can reach 4095. Saving an extra digit here or there is fairly meaningless in written formats, nor is it much of a difference in oral communication. However, when one communicates by gestures it is helpful to shorten the numbers where we can."`
+			label stormcontinue
+			`	"Now that you understand our numbers a bit better, you will be able to recognize that there is a lot of unexplained energy being transfered out there."`
+			choice
+				`	"Right. Is that what causes the red glow?"`
+					goto stormresume
+				`	"Thanks! I think that is all the details I can handle right now."`
```

Comment:
```suggestion
				`	"Thanks! I think that's all the details I can handle right now."`
```

Reply:
As per the previous comment regarding contractions, it is assumed that <first> is signing and thus not using contractions.